### DISCUSSION TITLE
Weird square shapes and text instead of icons

### DISCUSSION THREAD

**Discussion by yolobnb** [DNSCrypt/dnscrypt-proxy#2572] [Potential issues]
I have been having this issue for quite some time now. This happens on Google-related websites (Drive, Gmail, etc.). In place of where the icons should be, I see square shapes and weird characters. I will watch some images below. I have reinstalled the Unicode packages on my OS, but I need more hope.

![image](https://github.com/DNSCrypt/dnscrypt-proxy/assets/98680372/bff6b4b6-5e00-4c87-a950-93d1f225fd5b)

![image](https://github.com/DNSCrypt/dnscrypt-proxy/assets/98680372/7ed6e75e-c09c-4375-bcfa-4d1c6a244bcc)

![image](https://github.com/DNSCrypt/dnscrypt-proxy/assets/98680372/0d2dc6c6-ff86-4017-868e-0e0e5f1e76ad)

![image](https://github.com/DNSCrypt/dnscrypt-proxy/assets/98680372/621f6a7f-90bf-490c-bf71-506858bbe8d2)

![image](https://github.com/DNSCrypt/dnscrypt-proxy/assets/98680372/eb9e1d1d-43c8-49d6-a1d1-215a2aa3573f)

![image](https://github.com/DNSCrypt/dnscrypt-proxy/assets/98680372/5dcd22d1-6f1f-42ec-bbde-581dd55967f3)

This is due to the module. I have tested without it, and it works fine.  

Version: 2.1.5
Os: Arch (rolling) with Linux Zen (6.7.2-zen1-2-zen)

**Comment by ghost** [comment]
Looks like either your own blocklist or the DNS server you use blocks stuff like `fonts.googleapis.com` or `fonts.gstatic.com`.
You need to check and then either add the websites to your allowlist or report to the maintainer of the DNS server to unblock them.

**Reply by yolobnb** [reply]
This solved the issue. It was bothering me for a long time. Thanks for pointing out the domains.
